File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void showblock(int n);

int main(void)
{
    int n;
    do
    {
        n = get_int("Enter height of pyramid: ");
    }
    while (n < 1 || n > 8);

    showblock(n);
}

void showblock(int n)
{
    for (int i = 1; i <= n; i++)
    {
        for (int j = 0; j < n - i; j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
import cs50

# h = height
get_h = True

while get_h:
    h = cs50.get_int("Pyramid height: ")
    if h > 0 and h < 9:
        get_h = False
for i in range(h):
    print(" "*(h-i-1) + "#"*(i+1) + "  " + "#"*(i+1))
